### ISSUE TITLE
ImportError: cannot import name 'force_unicode'

### ISSUE THREAD

**Issue by danieleperugini** [Tivix/django-common#29]
I'm running django-common on django 1.9.1. (i changed util to utils) on python 3.5 but now i have this issue running scaffold command:

``` python
File "C:\Users\perug\PycharmProjects\myproj\myproj\env\lib\site-packages\django_common\admin.py", line 12, in <module>
    from django.utils.encoding import force_unicode
ImportError: cannot import name 'force_unicode'
```

I tried to downgrade to django 1.8.8 but it is still the same

**Comment by mjaworski** [comment]
Which version are you using? This shouldnt be an issue in the newer releases

**Comment by danieleperugini** [comment]
i'm using django-common-helpers 0.7.0 with django and python versions posted above

**Comment by mjaworski** [comment]
Some python3 compatibility fixes didnt make it into 0.7.0 we'll have a new release out for that soon.

**Comment by maxim-kht** [comment]
Fixed in https://github.com/Tivix/django-common/commit/978ea4b65502e53c84fdb0d25cc5ca9428382380
